metoncofit/DEA/Human1_mapping.m

```diff
@@ -0,0 +1,184 @@
+%% Import gene names and p values
```

Comment:
Logic is correct. Just turn this into a for loop, where the iterator are the sheets in the excel file. That'll pretty up this code.

Reply:
Eventually, you can combine these individual mapping scripts into a single `.mlx` file called `mapDataToCOBRA.mlx` and use the functions you wrote to execute mapping for any cobra model given the model, the list of genes in the data, and the data itself. This will be used as an example for other people who want to perform this.